CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ Add changes to the "Unreleased Changes" section. Once you create a version (and
 to a section with the version name.
 
 ## Unreleased Changes
+* Consider Is this Tax included in Basic Rate? checkbox in tax calculation per item.
```

Comment:
The fact that it's a checkbox is an implementation detail. We should be more abstract here, e.g. `Fix tax calculation for tax templates with tax included in basic rate`